phys/module_mp_fast_sbm.F

```diff
@@ -446,6 +447,200 @@ module module_mp_SBM_Collision
 
  contains
 ! +------------------------------------------------+
+subroutine coll_xyy_lwf_nlu (gx,gy,flx,fly,&
+      f_cw,dtime_coal,nkr,p_z,&
+      x,y, c,ima,prdkrn,indc)
+	implicit none
+
+	real(kind=r8size),intent(inout) :: gy(:),gx(:),fly(:),flx(:)
+	real(kind=r8size),intent(in) :: x(:),y(:),c(:,:)
+	integer,intent(in) :: ima(:,:)
+	real(kind=r8size),intent(in) :: prdkrn
+
+      real(kind=r8size), external :: f_cw
+      integer, intent(in) :: nkr
+      real(kind=r4size),intent(in) :: dtime_coal,p_z
+
+! ... Locals
+ real(kind=r8size) :: gmin,ckxy_ji,x01,x02,x03,gsi,gsj,gsk,gsi_w,gsj_w,gsk_w,gk,gk_w,&
+                      fl_gk,fl_gsk,flux,x1,flux_w,gy_k_w,gy_kp_old,gy_kp_w
+ integer :: j,jx0,jx1,i,iy0,iy1,jmin,indc,k,kp
+! ... Locals
+
+	  gmin = 1.0d-60
+
+! jx0 - lower limit of integration by j
+do j=1,nkr-1
+   jx0=j
+   if(gx(j).gt.gmin) goto 2000
```

Comment:
In this case, you can use the `EXIT` statement to exit the loop, as in `if(gx(j).gt.gmin) exit`. Fortran's `CYCLE`/`EXIT` correspond to C's `continue`/`break`.